.github/workflows/mariadb.yml

```diff
@@ -0,0 +1,122 @@
+name: MariaDB
+on:
+  push:
+    paths:
+      - "**/*.cpp"
+      - "**/*.h"
+      - "**/*.hpp"
+      - "**/CMakeLists.txt"
+      - "**/Makefile.am"
+      - ".github/workflows/mariadb.yml"
+      - "configure.ac"
+      - "doc/**"
+      - "mysql-test/**"
+      - "packages/**"
+      - "version_*"
+  pull_request:
+    paths:
+      - "**/*.cpp"
+      - "**/*.h"
+      - "**/*.hpp"
+      - "**/CMakeLists.txt"
+      - "**/Makefile.am"
+      - ".github/workflows/mariadb.yml"
+      - "configure.ac"
+      - "doc/**"
+      - "mysql-test/**"
+      - "packages/**"
+      - "version_*"
+  schedule:
+    - cron: |
+        0 0 * * *
+concurrency:
+  group: ${{ github.head_ref || github.sha }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  build:
+    name: Build
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ruby
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb
+          repository: MariaDB/server
+      - name: Remove bundled Mroonga
+        run: |
+          rm -rf mariadb/extra/groonga
+          rm -rf mariadb/storage/mroonga
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/extra/groonga
+          repository: groonga/groonga
+          submodules: recursive
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/storage/mroonga
+          repository: mroonga/mroonga
+          submodules: recursive
+      - name: Build MariaDB
+        run: |
+          cmake \
+            -Smariadb \
+            -Bmariadb.build \
+            -G Ninja \
+            -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Would you need this?
Because the purpose of this CI is build test.
I think we don't execute debug on CI.

Reply:
In general, `-DCMAKE_BUILD_TYPE=Debug` is better for running tests. If some tests are crashed, we can get backtrace with `-DCMAKE_BUILD_TYPE=Debug`.